janitor/functions.py

```diff
@@ -17,6 +17,8 @@
     Tuple,
     Union,
 )
+from types import SimpleNamespace
```

Comment:
Is this import related to the current PR?

Reply:
Good catch!

Reply:
This was an unused import.